apply-notch-hax.sh

```diff
@@ -7,8 +7,24 @@ if [ $UID -eq 0 ]; then
 fi
 
 WORK="$HOME/.cache/unity8-notch-hax"
-DEVICE="$(getprop ro.product.device)" # e.g. 'yggdrasil'
+DEVICE="$1"
+
+if [ -z $DEVICE ]; then
+	echo "Attempting to detect device name"
```

Comment:
This is a purely visual thing, but could each of the added `echo`'s to the script have a `>> ` prefix like the existing ones? The message can also include why it's attempting to detect the device name: `No device specified, attempting to detect device name...`